infra/docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# ai-vocabulary-builder Dockerfile
```

Comment:
I prefer the Dockerfile in the main branch because its primary purpose is to build a runnable image from source files. 

This version is more of a shortcut for users who don't have a Python environment, which is less common.